### PR TITLE
Ghost role spawners for mappers

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Lets you make a custom landmark (gear, mob name, species, etc) that will spawn a ghost as a mob and give it some flavour text as to its new role.

Intended for use in stuff like away missions, space ruins, etc, if you want to have a player be a lost astronaut or strange hermit or whatever.

TODO: More comments explaining stuff, use the mob name var again, make a few example landmarks.

### CODE REVIEW

**Comment by WJohn** [comment]
MIIIGRAAAANTS!

**Comment by Zilenan91** [comment]
Alien invasions? Syndicate invasions? Wizard invasions? Dungeon Masters? So many possibilities!

**Comment by Tokiko1** [comment]
An option to have the mob take a random name and appearance or use the ghost's appearance preferences would be nice for things where you want some mob variety, like prisoners.

File: code/modules/awaymissions/corpse.dm [hunk 1]
```diff
@@ -276,3 +291,17 @@
...
+
+/obj/effect/landmark/corpse/commander/alive
+	death = FALSE
+	roundstart = FALSE
+	name = "sleeper"
+	icon = 'icons/obj/Cryogenic2.dmi'
+	icon_state = "sleeper"
+	flavour_text = "You are a Nanotrasen Commander!"
+
+/obj/effect/landmark/corpse/attack_ghost(mob/user)
```
Comment (Anonus) [inline code/modules/awaymissions/corpse.dm]: Er, shouldn't the invisibility on these be low enough so ghosts CAN click them?

**Pull request merged**